ui/components/multichain/edit-accounts-modal/edit-accounts-modal.test.tsx

```diff
@@ -40,10 +41,21 @@ const render = (
     mockState.metamask.internalAccounts.accounts,
   ) as unknown as MergedInternalAccount[];
 
+  const accountsWithCaipAccountId = accounts.map((account) => {
+    const { namespace, reference } = parseCaipChainId(account.scopes[0]);
```

Comment:
Are we safe to only use the first scope here? If so, why?

Reply:
in this particular test file, yes. In general, it's okay depending on the scenario for now. However this will need adjustment in a few places for SCAs